vmdb/app/models/miq_server/role_management.rb

```diff
@@ -36,6 +38,10 @@ def sync_active_roles
     @active_role_names = self.active_role_names
   end
 
+  def apache_needed?
+    !(active_role_names & ROLES_NEEDING_APACHE).empty?
```

Comment:
Another option is the `include_any?` extension method from more_core_extensions..

```ruby
active_role_names.include_any?(ROLES_NEEDING_APACHE)
```